src/components/Footer/Footer.jsx

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { FooterContainer, FooterLogo, FooterMenu } from './Footer.styles';
 
-const Footer = () => <div>Footer Component</div>;
+const Footer = () => (
+  <FooterContainer>
+    <div>
+      <FooterLogo>
+        <img
+          src='https://www.themoviedb.org/assets/2/v4/logos/v2/blue_square_2-d537fb228cf3ded904ef09b136fe3fec72548ebc1fea3fbbd1ad9e36364db38b.svg'
```

Comment:
please download all svg images to the assets and use them  as components

Reply:
done